website/src/utils/amplitude.ts

```diff
@@ -1,9 +1,17 @@
+import { parse } from 'bowser';
 import amplitudeHttp from './amplitude-client';
 
 export const initAmplitude = () => {
   const apiKey = '1e1d36fa96573d0839c6c3ccaffb7f62';
+  const systemInfo = parse(window.navigator.userAgent);
+  const language = window.navigator.language;
 
-  amplitudeHttp.init(apiKey);
+  amplitudeHttp.init(apiKey).sendUserProperties({
+    browser: `${systemInfo.browser.name} ${systemInfo.browser.version}`,
+    OS: `${systemInfo.os.name} ${systemInfo.os.versionName} v${systemInfo.os.version}`,
+    platform: `${systemInfo.platform.vendor} ${systemInfo.platform.type}`,
+    language,
```

Comment:
Are all of that  data properties being send mentioned in Semrush legal notes?

Reply:
yes :)

Reply:
almost :see_no_evil: